src/pyfaf/storage/report.py

```diff
@@ -175,9 +175,9 @@ def frames(self):
     def normalized(self):
```

Comment:
You can remove the function completely as it isnt used anywhere and `btp_thread()` was removed in commit b79e74351a85ccbabde565f2397d36b4629f0b00